openspec/changes/standardize-jsdoc-tracking/tasks.md

```
@@ -0,0 +1,18 @@
+1. **Create a canonical docs home for JSDoc trackers.**
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)